src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -196,11 +205,57 @@ private void linkToCase(User defendantUser, User anonymousCaseWorker, String let
             letterHolderId
         );
 
-        coreCaseDataService.update(
+        String defendantEmail = defendantUser.getUserDetails().getEmail();
+        CaseDetails caseDetails = coreCaseDataService.update(
             defendantUser.getAuthorisation(),
-            CCDCase.builder().id(Long.valueOf(caseId)).defendantId(defendantId).build(),
+            CCDCase.builder()
+                .id(Long.valueOf(caseId))
+                .defendantId(defendantId)
+                .defendantEmail(defendantEmail)
+                .build(),
             CaseEvent.LINK_DEFENDANT
         );
+
+        scheduleEmailNotificationsForDefendantResponse(caseId, defendantId, defendantEmail, caseDetails);
+    }
+
+    private void scheduleEmailNotificationsForDefendantResponse(
```

Comment:
This needs to happen in the claim service,
this story is not only about CCD it should happen whether or not ccd is on